docs/contributing/HACKING.md

```diff
@@ -94,12 +94,15 @@ If you installed HLS in the old, complicated way, you can safely remove it.  HLS
 
 You should also set the `FOSSA_SKIP_EMBED_FILE_IN_HLS` environment variable for HLS. This prevents HLS from embedding binaries, which helps to avoid a giant memory footprint for HLS.
 
+You can also tell the Fourmolu plugin to use an external config. This ensures that it picks up our `fourmolu.yaml` file.
```

Comment:
I know it's not related to this PR, but I thought I'd sneak this in while I had it in my head